test/externalTests/common.sh

```diff
@@ -24,8 +24,8 @@ set -e
 
 function verify_input
 {
-    if [ ! -f "$1" ]; then
-        printError "Usage: $0 <path to soljson.js>"
+    if [ -z "$1" ] || [ -z "$2" ]; then
```

Comment:
```suggestion
    if [ -z "$1" ] || [! -f $1] || [ -z "$2" ]; then
```

Reply:
Ah, true. Of course we need to check if the path exists.